allennlp/modules/stacked_alternating_lstm.py

```diff
@@ -106,5 +106,5 @@ def forward(self,  # pylint: disable=arguments-differ
             output_sequence, final_state = layer(output_sequence, state)
             final_states.append(final_state)
 
-        final_state_tuple = (torch.cat(state_list, 0) for state_list in zip(*final_states))
+        final_state_tuple = tuple(torch.cat(state_list, 0) for state_list in zip(*final_states))
```

Comment:
Could you please add a return type on line 74 and correct the `Returns` section of the docstring?

Reply:
Done, thanks.